build/css/application.css

```diff
@@ -4799,6 +4799,36 @@ blockquote p {
 .header_cases h1 {
```

Comment:
we have to remove build folder from git

Reply:
Okay, maybe then I will add these folders to `.gitignore` to prevent adding in future.